### PR TITLE
Remove border bottom header wrapper

### PR DESCRIPTION
**Pull request by pochretien**
add uts
add example

image

image

### CODE REVIEW

File: src/utils/ComponentUtils.ts [hunk 1]
```diff
@@ -15,3 +16,19 @@ export const DisplayClass = {
 };
 
 export type ComponentContent = (new () => React.Component) | string | (() => JSX.Element);
+
+export const getBasicDocumentLink = (url: string, title: string = ''): ILinkSvgProps => {
```
Comment (GermainBergeron) [inline src/utils/ComponentUtils.ts]: Is this used?

**Comment by GermainBergeron** [comment]
Your build failled: 

```
WARNING in ./src/utils/ComponentUtils.ts
[33, 4]: Missing semicolon
[32, 6]: Missing trailing comma
```

File: src/utils/ComponentUtils.ts (hunk 1 above)
Comment (pochretien) [reply src/utils/ComponentUtils.ts]: Yep, its only an util to create my link faster

Comment (MichelML) [reply src/utils/ComponentUtils.ts]: this will surely be useful 👍 thanks

**Pull request merged**